vw_golf_mk4.dbc

```diff
@@ -770,25 +770,19 @@ BO_ 1340 Fahrwerk_1: 1 XXX
  SG_ Ansteuererung_Fahrzeugniveau : 0|4@1+ (1,0) [0|15] "" XXX
 
 BO_ 1472 EPB_1: 8 XXX
- SG_ CHECKSUM : 56|8@1+ (1,0) [0|255] "" XXX
  SG_ COUNTER : 0|4@1+ (1,0) [0|15] "" XXX
- SG_ EP1__Text : 52|4@1+ (1,0) [0|8] "" Vector__XXX
- SG_ EP1_Failure_gelb : 51|1@1+ (1,0) [0|1] "" XXX
- SG_ EP1_Failure_BKL : 50|1@1+ (1,0) [0|1] "" XXX
- SG_ EP1_Warnton : 49|1@1+ (1,0) [0|1] "" XXX
- SG_ EP1_Fkt_Lampe : 48|1@1+ (1,0) [0|1] "" Vector__XXX
- SG_ EP1_Freigabe_Ver : 33|1@1+ (1,0) [0|1] "" XXX
- SG_ EP1_Failureeintr : 32|1@1+ (1,0) [0|1] "" XXX
- SG_ EP1_Neig_winkel : 16|8@1+ (1,-128) [-128|127] "g" XXX
- SG_ EP1_Verzoegerung : 24|8@1+ (0.048,-7.968) [-7.968|4.224] "g" XXX
- SG_ EP1_Sta_NWS : 15|1@1+ (1,0) [0|1] "" XXX
- SG_ EP1_Schalterinfo : 13|2@1+ (1,0) [0|3] "" XXX
- SG_ EP1_Spannkraft : 8|5@1+ (1,0) [0|30] "kN" XXX
- SG_ EP1_Sta_Schalter : 7|1@1+ (1,0) [0|1] "" XXX
+ SG_ EP1_Fehler_Sta : 4|2@1+ (1,0) [0|3] "" Vector__XXX
  SG_ EP1_Sta_EPB : 6|1@1+ (1,0) [0|1] "" XXX
- SG_ EP1_Failure_Sta : 4|2@1+ (1,0) [0|3] "" XXX
+ SG_ EP1_Sta_Schalter : 7|1@1+ (1,0) [0|1] "" XXX
```

Comment:
can you revert the sorting? it's not trivial to review and there's nothing guaranteeing it in the future

Reply:
I asked him to do it. This DBC is kind of trash (Golf Mk4 was like 1998-2005) and we're slowly bringing in some elements from more recent data sources. The new ones, as well as the MQB DBC, are all properly sorted.